[resolved] Error when compiling from sources

Hi,

I can't compile MinIO from sources for the last 2 versions 2020-09-05T07-14-49Z and 2020-09-08T23-05-18Z.

When running "go install" it fails with these errors:
```
pkg/certs/certs.go:297:18: hello.SupportsCertificate undefined (type *tls.ClientHelloInfo has no field or method SupportsCertificate)
pkg/certs/certs.go:331:20: reqInfo.SupportsCertificate undefined (type *tls.CertificateRequestInfo has no field or method SupportsCertificate)
```

It could be related to https://github.com/caddyserver/caddy/issues/3326 and the use of Go < 1.14
The problem is that Go 1.14 is not supported by MinIO: https://github.com/minio/minio/blob/RELEASE.2020-09-08T23-05-18Z/go.mod#L3


Here is a simple way to reproduce the problem with `docker run -it alpine:3.12`:
```bash
apk add go git build-base

git clone https://github.com/minio/minio
cd minio
git checkout tags/RELEASE.2020-09-08T23-05-18Z

go install
```

Is it possible to bump Go to 1.14 or downgrade dependencies that require Go 1.13?

--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/minio/minio
 
-go 1.13
+go 1.14
 
 require (
 	cloud.google.com/go v0.39.0

